### PR TITLE
Update index.adoc

### PR DESCRIPTION
**Pull request by rimshach**
A new page having information about endpoint-dsl has been added.This page shall be accessed from user manual.

### CODE REVIEW

File: docs/user-manual/modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -14,6 +14,8 @@ routing rules in your IDE, whether in a Java or XML editor.
 
 For a deeper and better understanding of Apache Camel, an xref:faq/what-is-camel.adoc[Introduction] is provided.
 
+Furthermore, camel has added a new API i.e. Endpoint-dsl. More information can be founded under documentation section.
```
Comment (oscerd) [inline docs/user-manual/modules/ROOT/pages/index.adoc]: "Can be found" should be better I guess

Comment (rimshach) [reply docs/user-manual/modules/ROOT/pages/index.adoc]: Yes, this should be "can be found".I'll fix it

Comment (rimshach) [reply docs/user-manual/modules/ROOT/pages/index.adoc]: I have made the change in the file

File: docs/user-manual/modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -14,6 +14,8 @@ routing rules in your IDE, whether in a Java or XML editor.
 
 For a deeper and better understanding of Apache Camel, an xref:faq/what-is-camel.adoc[Introduction] is provided.
 
+Furthermore, camel has added a new API i.e. Endpoint-dsl. More information can be found under documentation section.
```
Comment (oscerd) [inline docs/user-manual/modules/ROOT/pages/index.adoc]: Looking at this better, I don't think this is the right place for this. But we could move it later

**Comment by davsclaus** [comment]
Yeah its IMHO not the right place. It should be added as a link from the user manual page: https://camel.apache.org/manual/latest/

**Comment by rimshach** [comment]
I'll remove it from here,but under what heading in user manual should the link come?

**Comment by rimshach** [comment]
The link is currently under documentation heading

**Pull request merged**